ServerCore/Pages/Teams/List.cshtml

```diff
@@ -16,25 +16,25 @@
             Registration will be open soon.
         </div>
     }
-    else if (DateTime.UtcNow > Model.Event.TeamRegistrationBegin)
+    else if (DateTime.UtcNow > Model.Event.TeamRegistrationEnd)
     {
         <h2>Registration closed</h2>
         <div>
-            You are not part of a team for this event, and unfortuantely we're no longer accepting new teams into the event. :(
+            You are not part of a team for this event, and unfortuantely we're no longer accepting new teams into the event. You can still request to join an existing team.
```

Comment:
We should only show the "you can still join" conditionally, only when the event is still accepting team changes. Not going to block on it for opening registration though.

Reply:
Added :)